CHANGES-DNF-5.md

```diff
@@ -49,6 +49,14 @@ options and implemented only for related commands
 Renaming boolean options to format `--<option>`, and `--no-<option>`
 `--nobest` -> `--no-best`. Proposing to keep `--nobest` as a deprecated option.
 
+### --strict deprecation
+`--strict` option is now deprecated. The problem with this option is that it does two things:
```

Comment:
If I understand correctly I would maybe reword it a bit:
- isn't the `strict` option a config option? I can't find any `--strict` switch
- I would say if the strict option was disabled it allows those two things, currently to me the text suggests that if the option was enabled it allows those things.

Reply:
Thanks for suggestions. I've tried to make it more clear.